src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  list-style: none;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+}
+
+.nav__link {
+  position: relative;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+}
+
+.hover {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  left: 0;
+  bottom: -23px;
```

```suggestion
  bottom: 0;
```
link should be full height